packages/web/src/common/clients/wallet-client/transaction-messages/common.ts

```diff
@@ -150,8 +150,8 @@ export async function makeTransactionMessagesWithApproves(
 
     const previousAmount = accumulated[current.tokenPath][current.targetAddress].amount || "0";
     const sumAmountBN = BigNumber(previousAmount).plus(current.amount.toString());
-    if (sumAmountBN.isGreaterThan(MAX_UINT64.toString())) {
-      accumulated[current.tokenPath][current.targetAddress].amount = MAX_UINT64.toString();
+    if (sumAmountBN.isGreaterThan(MAX_INT64.toString())) {
+      accumulated[current.tokenPath][current.targetAddress].amount = MAX_INT64.toString();
```

Comment:
how about storing the result of `MAX_INT64.to_string()` in a variable to reduce unnecessary function calls and conversions? 

The practical approach would be to make it a global variable or constant.